coalib/results/Diff.py

```diff
@@ -351,14 +351,22 @@ def __bool__(self):
     def delete_line(self, line_nr):
         """
         Mark the given line nr as deleted. The first line is line number 1.
+
+        Raises an exception if line number doesn't exist in the diff.
         """
+        if line_nr > len(self._file):
+            raise IndexError('The given line number is out of bounds')
```

Comment:
Use a period at the end? Just suggesting because the TypeError above has a period at the end :P